CMakeLists.txt

```diff
@@ -139,6 +139,7 @@ if (NS_BTLA_UT)
 endif()
 include(FindOpenMP)
 
+set(BTLA_USE_OPENMP ON)
```

Comment:
will this change affect your thread-pool for hybrid CPUs? @yuchengliu1

Reply:
It's not necessary to set OPENMP here. BESTLA default to set it on in its own `CMakeLists.txt`.

Reply:
> It's not necessary to set OPENMP here. BESTLA default to set it on in its own `CMakeLists.txt`.

has been changed now https://github.com/intel/neural-speed/blob/e2d365206fea12b66bd2f9d7c921d2ad9de3a439/bestla/CMakeLists.txt#L7